.circleci/config.yml

```diff
@@ -32,24 +34,29 @@ jobs:
       - run:
           name:  "Test: Execute Python OWASP ZAP Vulnerability Scan"
           command: "cd tdrs-backend; ./zap-scanner.sh"
+      - store_artifacts:
+          path: tdrs-backend/reports/owasp_report.html
   test-frontend:
     working_directory: ~/tdp-apps
-    docker:
-      - image: cypress/base:10
-        user : root
+    machine:
+      docker_layer_caching: false
     steps:
-      - setup_remote_docker:
-         docker_layer_caching: false
       - checkout
-      - run:
-          name: 'Ensure docker exists,otherwise install it.'
-          command: "./scripts/docker-check.sh"
       - run:
           name: 'Ensure docker-compose exists,otherwise install it.'
           command: "./scripts/docker-compose-check.sh"
-      - run:
-          name: "Install Package Dependencies"
-          command: "cd tdrs-frontend; yarn"
+      - run: 
```

Comment:
This was previously handled by the `cypress` base image we were using - but we cannot use that with the `machine executor`.